src/jdk.jdwp.agent/windows/native/libjdwp/linker_md.c

```diff
@@ -22,6 +22,12 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
+ 
```

Comment:
Please remove the trailing whitespace.

Reply:
Will do.

Reply:
Seems unnecessary, and contrary to standard practice. We can resolve this across all extensions files in a separate PR if this turns out to be a priority.